### ISSUE TITLE
Add packages

### ISSUE THREAD

**Issue by marcosvega91** [smeijer/testing-playground#34]
The project step by step will grow and could have in the next future more packages:

1. site
2. embedded
3. chrome extension
4. devtools extension

We should think to add something similar to [lerna](https://github.com/lerna/lerna) to manage all of them

**Comment by aganglada** [comment]
@marcosvega91 NPM released workspaces a few weeks ago, I can. spend some time giving it a shot, would you like me to do so? https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/

**Comment by smeijer** [comment]
@aganglada, can you tell me what the impact is for contributions/maintainers?

**Comment by aganglada** [comment]
@smeijer Depends of what you mean with impact, but basically:

* We will structure the repo in different packages `app`, `chrome-extension`, `devtools`
* We will need to change contributing guidelines so they follow the new structure
* Change on documentation
* Change on github actions to make sure it all works like it does currently.

Is that what you mean?

**Comment by smeijer** [comment]
Thanks,

That. And also; once it has been restructured. What do I need to do to install a package into `devtools`, and what do I need to install it into `app`?

But please, feel free to give it a shot. It would be much appreciated. :heart:

**Comment by aganglada** [comment]
@smeijer yep, that will be something like `npm workspace app add your-package`. I will investigate 😄
